ui/app/components/app/gas-customization/gas-modal-page-container/gas-modal-page-container.container.js

```diff
@@ -196,18 +232,26 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
     ...otherDispatchProps,
     ...ownProps,
     onSubmit: (gasLimit, gasPrice) => {
-      if (isConfirm) {
-        dispatchUpdateConfirmTxGasAndCalculate(gasLimit, gasPrice)
-        dispatchHideModal()
-      } else if (isSpeedUp) {
-        dispatchCreateSpeedUpTransaction(txId, gasPrice)
-        dispatchHideSidebar()
-        dispatchCancelAndClose()
-      } else {
-        dispatchSetGasData(gasLimit, gasPrice)
-        dispatchHideGasButtonGroup()
-        dispatchCancelAndClose()
-      }
+        if (isConfirm) {
```

Comment:
Nit: this whole fn has been over-indented. Can we de-indent this one level to correct the diff?